src/main/java/uk/gov/hmcts/reform/civil/helpers/sdo/SdoHelper.java

```diff
@@ -96,11 +97,51 @@ public static boolean hasSmallAdditionalDirections(CaseData caseData, String add
     public static String getSmallClaimsHearingTimeLabel(CaseData caseData) {
         SmallClaimsHearing smallClaimHearing = caseData.getSmallClaimsHearing();
 
+        String hearingTimeEstimateLabel = "";
+
         if (smallClaimHearing != null) {
-            return smallClaimHearing.getTime().getLabel().toLowerCase(Locale.ROOT);
+            if (smallClaimHearing.getTime().getLabel() != null && smallClaimHearing.getTime().getLabel().equals("Other")) {
```

Comment:
if you flip the equals you can remove the null check and coverage will have one less branch to care about